dask/array/tests/test_overlap.py

```diff
@@ -294,6 +294,11 @@ def test_asymmetric_overlap_boundary_exception():
 
 
 def test_map_overlap():
+    x = da.from_array(np.arange(10), chunks=5)
```

Comment:
nitpick, but any reason not to use 
```suggestion
    x = da.arange(10, chunks=5)
```
as the rest of the checks in this test do?

Reply:
Also probably it'd be better to pull this out into a new test with an explicit name. Something like: `test_map_overlap_escapes_to_map_blocks_when_depth_is_zero`